### PR TITLE
Add meta tags

### PR DESCRIPTION
**Pull request by cveneziani**
- Add Images, Meta Tags and URL helpers
- Add description to all pages
- Namespace images per page path

### CODE REVIEW

File: source/layouts/shared/_meta_tags.html.slim [hunk 1]
```diff
@@ -0,0 +1,35 @@
+meta charset="utf-8"
+meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"
+meta name="viewport"              content="width=device-width, initial-scale=1.0"
+
+meta name="description"           content=meta_tag_description
```
Comment (flexbox) [inline source/layouts/shared/_meta_tags.html.slim]: `content="#{meta_tag_description}"`

File: source/layouts/shared/_meta_tags.html.slim [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+/ Geocoding
+meta name="geo.placename"         content="Lille, Nord"
+meta name="geo.position"          content="50.629250;3.057256"
+meta name="geo.region"            content="FR-59"
+
+/ Microdata
+meta itemprop="name"              content=meta_tag_title
```
Comment (flexbox) [inline source/layouts/shared/_meta_tags.html.slim]: same as above

File: source/layouts/shared/_meta_tags.html.slim (hunk 1 above)
Comment (flexbox) [inline source/layouts/shared/_meta_tags.html.slim]: why the `title` is not here ?

Comment (cveneziani) [inline source/layouts/shared/_meta_tags.html.slim]: Because it's not a `meta` tag? :grimacing:

**Pull request merged**